### PR TITLE
jck: hardcode jck.env.runtime.url.ftpURL value

### PR DESCRIPTION
**Pull request by gdams**
without this we get: `Configuration file can not be imported due to question with specified tag "jck.env.runtime.url.httpURL" was not found on path. The path is:jck.intro`

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
@gdams , could you share the name of the test target that this fixes?

**Comment by gdams** [comment]
@Mesbah-Alam it fixes tests in `jck-runtime-api-java_net_0`. The fix is actually carried over from the old systemtest repo: https://github.com/adoptium/aqa-systemtest/blob/master/openjdk.test.jck/config/jck8c/runtime.jti#L199

File: jck/jck.mk [hunk 1]
```diff
@@ -62,7 +62,7 @@ ifndef JCK_ROOT
 endif
 
 ifndef CONFIG_ALT_PATH
-  export CONFIG_ALT_PATH:=$(TEST_ROOT)$(D)jck$(D)/jtrunner/config/default
+  export CONFIG_ALT_PATH:=$(TEST_ROOT)$(D)jck$(D)jtrunner/config/default
```
Comment (Mesbah-Alam) [inline jck/jck.mk:65]: Thanks for this update. Since it's being fixed here, could we get all the `/` replaced with `$(D)` in `CONFIG_ALT_PATH` ?

**Review by Mesbah-Alam — APPROVED** [review]
LGTM

File: jck/jtrunner/JavaTestRunner.java [hunk 2]
```diff
@@ -224,13 +224,13 @@ public static boolean generateJTB() throws Exception {
...
-		jtliteJarFullPath = jckRoot + File.separator + testSuiteFolder + File.separator + "lib" + File.separator + "jtlite.jar"; 
-		classesFullPath = jckRoot + File.separator + testSuiteFolder + File.separator + "classes";
+		jckPolicyFileFullPath = jckBase + File.separator + "lib" + File.separator + "jck.policy";
+		javatestJarFullPath = jckBase + File.separator + "lib" + File.separator + "javatest.jar";
+		jtliteJarFullPath = jckBase + File.separator + "lib" + File.separator + "jtlite.jar"; 
+		classesFullPath = jckBase + File.separator + "classes";
 		nativesLoc = jckRoot + File.separator + "natives" + File.separator + platform;
 		jtiFile = testRoot + File.separator + "jck" + File.separator + "jtrunner" + File.separator + CONFIG + File.separator + jckVersion + File.separator + testSuite.toLowerCase() + ".jti"; 
-		fileUrl = "file:///" + testSuiteFolder + "/testsuite.jtt";
+		fileUrl = "file:///" + jckBase + "/testsuite.jtt";
```
Comment (sophia-guo) [inline jck/jtrunner/JavaTestRunner.java:233]: Is this change on purpose? As @gdams mentioned this PR will hit `Unexpected exception:java.io.FileNotFoundException: /JCK-runtime-8c/testsuite.jtt (No such file or directory)` would suggest to change back to `testSuiteFolder`.

Comment (gdams) [reply jck/jtrunner/JavaTestRunner.java:233]: This change is on purpose, currently the directory cannot be found and fails. This uses the correct test directory

Comment (Mesbah-Alam) [reply jck/jtrunner/JavaTestRunner.java:233]: Thanks for spotting this @gdams!

**Review by smlambert — APPROVED** [review]
